testutils/networksuite/networksuite.go

```diff
@@ -81,6 +83,8 @@ func (nts *NetworkTestSuite) SetupSuite() {
 	// initialize genesis
 	require.NoError(nts.T(), cfg.Codec.UnmarshalJSON(cfg.GenesisState[auctiontypes.ModuleName], &nts.AuctionState))
 	nts.AuctionState = populateAuction(r, nts.AuctionState)
+	nts.AuctionEscrow = account.NewAccount()
+	nts.AuctionState.Params.EscrowAccountAddress = nts.AuctionEscrow.Address().Bytes()
```

Comment:
why r we changing the escrow addy?

Reply:
For convenience of having an account object. I'll change it back because it does add some non-zero complication.